com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardProgram.hlsl

```diff
@@ -466,6 +466,14 @@ half4 PixelStage(Varyings input, bool facing : SV_IsFrontFace) : SV_Target
 #endif 
 #endif
 
+#if defined(_EMISSION)
+#if defined(_URP)
+    half3 emissionMap = SAMPLE_TEXTURE2D(_EmissiveMap, sampler_EmissiveMap, input.uv).xyz;
```

Comment:
Could you update the [meta shader](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/385952c3e0c59b934f7846cb2b815977cb22dcda/com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardMetaProgram.hlsl#L78) to also sample this emissive map? That way when generating static lighting (light maps) this texture is considered.